source/wp-content/themes/wporg-news-2021/sass/base/_layout.scss

```diff
@@ -1,14 +1,26 @@
 // Setup the main container to use a flex box-model
 // allowing us to simplify the layout and "sticky" bar.
 .wp-site-blocks {
-	min-height: 100vh;
 	display: flex;
 	flex-direction: column;
 	margin-top: -(var(--wp-admin--admin-bar--height, 0));
+}
+
+// Make sure the page grows to fill the browser window, and that the footer is stuck to the bottom.
+html,
+body,
+.wp-site-blocks {
+	height: 100%;
```

Comment:
This messes with the sticky header unfortunately - I had to fix a similar issue on translate (part of https://github.com/WordPress/wporg-mu-plugins/pull/96). You can test it by trying this out on a long page, once you scroll past the viewport height, the header unsticks.

Reply:
👍🏻 , switching back to draft while I work on that

Reply:
I think that's solved in 5104cebbb7654842bc1c819fb90aea47355bf697